index.html

```diff
@@ -5,7 +5,7 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <link rel="stylesheet" href="style.css" />
-    <title>Porfolio Website</title>
+    <title>Portfolio Website</title>
```

Comment:
- [ ] There is a horizontal scroll at the bottom of the page. Please kindly remove it also 👌🏼👋🏼👍🏼

![image](https://user-images.githubusercontent.com/69011963/172344664-16b29d1d-e033-4743-be49-9880d4791a02.png)

Reply:
@Rustamxon7 Thank you for reviewing my project, I will make the necessary corrections as suggested